Contribute/how-to-write-links.md

```diff
@@ -57,42 +77,269 @@ To create an inline link from a Docs technical article to another Docs technical
   `[link text](../directory/article-name.md)`
 
 > [!NOTE]
-> None of the previous examples use the `~/` as part of the link. To link to an absolute path that begins at the root of the repository, start the link with `/`. Including the `~/` produces invalid links when navigating the source repositories on GitHub. Starting the path with `/` resolves correctly.
+> None of the previous examples use the `~/` as part of the link. To link to an absolute path that
+> begins at the root of the repository, start the link with `/`. Including the `~/` produces invalid
+> links when navigating the source repositories on GitHub. Starting the path with `/` resolves
+> correctly.
+
+### Structure of links on docs.microsoft.com
+
+Content published on docs.microsoft.com has the following URL structure:
+
+```
+https://docs.microsoft.com/<locale>/<product-service>/[<feature-service>]/[<subfolder>]/<topic>[?view=<view-name>]
+```
+
+Examples:
 
-To link to an article in a different docset, even if the file is in the same repository, use the following syntax:
+```
+https://docs.microsoft.com/en-us/azure/load-balancer/load-balancer-overview
+https://docs.microsoft.com/en-us/powershell/azure/overview?view=azurermps-5.1.1
+```
 
-`[link text](/docset-root/directory/article-name)`
-   
-For example, if an article whose root URL is `https://docs.microsoft.com/dotnet` links to an article whose root URL is `https://docs.microsoft.com/visualstudio`, the link would look like `[link text](/visualstudio/directory/article-name)`.
+- `<locale>` - identifies the language of the article (example: en-us or de-de)
+- `<product-service>` - the name of the product or service (example: powershell, dotnet, or azure)
+- `[<feature-service>]` - (optional) the name of the product's feature or subservice (example:
+  csharp or load-balancer)
+- `[<subfolder>]` - (optional) the name of a subfolder within a feature
+- `<topic>` - the name of the article file for the topic (example: load-balancer-overview or
+  overview)
+- `[?view=\<view-name>]` - (optional) the view name used by the version selector for content that
+  has multiple versions available (example: azps-3.5.0)
 
 > [!TIP]
-> Articles in the same *docset* have the same URL fragment after "docs.microsoft.com". For example, `https://docs.microsoft.com/dotnet/core/get-started` and `https://docs.microsoft.com/dotnet/framework/install` are in the same docset, and `https://docs.microsoft.com/dotnet/core/get-started` and `https://docs.microsoft.com/visualstudio/whats-new` are in different docsets.
 
-## Links to anchors
+> In most cases, articles in the same _docset_ have the same `<product-service>` URL fragment. For
```

Comment:
Because there are exceptions. There are docsets in other repos that publish under https://docs.microsoft.com/azure. For example https://github.com/microsoftDocs/azure-dev-docs-pr contains multiple docsets that all publish under the `azure` URL. 

For the public contributor, it is too confusing to explain the corner cases, and how to figure it out. If a public contributor gets the URL wrong in the PR we should be able to suggest fixes for those corner cases.

Reply:
Fixed the TIP block